sdk/communication/AzureCommunication/Source/Authentication/CommunicationAuthenticationPolicy.swift

```diff
@@ -28,45 +28,33 @@
 import AzureCore
 #endif
 import Foundation
-/**
- The Azure Communication Services authentication policy.
- */
-public class CommunicationUserCredentialPolicy: Authenticating {
-    public var next: PipelineStage?
 
+public class CommunicationPolicyTokenCredential: TokenCredential {    
```

Comment:
Should the file renamed to CommunicationPolicyTokenCredential.swift?

Reply:
Yes I should update the name to the file good catch